### PR TITLE
Fixed some linting errors

### PR DESCRIPTION
**Pull request by tanmayanand44**
## What did you implement:

Fixed some linting errors.

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [ ] Fix linting errors
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config / commands / resources
- [ ] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** Yes
***Is it a breaking change?:*** NO

### CODE REVIEW

File: test/compute/ec2.js [hunk 1]
```diff
@@ -20,235 +20,235 @@ describe('AWS EC2', () => {
...
-        [
-          'Transfer-Encoding',
-          'chunked',
-          'Date',
-          'Sat, 15 Jul 2017 13:58:34 GMT',
-          'Server',
-          'AmazonEC2',
-        ]
-      );
+ec2.list(params).then((res) => {
```
Comment (rajikaimal) [inline test/compute/ec2.js]: Can you double check the indentation here.

**Comment by rajikaimal** [comment]
@tanmayanand44 

Can you check the indentation again, seems like indentation of `describe` block and `it` blocks has the same indentation levels, which should not happen. `it` should be an inner block to `describe` block.

**Review by rajikaimal — CHANGES_REQUESTED** [review]
Refer comments section of the PR.

**Comment by tanmayanand44** [comment]
@rajikaimal Please check now.Done as per your review.

**Comment by rajikaimal** [comment]
Looks good !

**Pull request merged**